controllers/ansibleee_controller.go

```diff
@@ -72,6 +72,47 @@ func (r *AnsibleEEReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	//configMapVars := make(map[string]env.Setter)
 	//r.configMapForAnsibleEE(ctx, instance, helper, &configMapVars)
 
+	if instance.Spec.EnablePlugin == true {
+		if len(instance.Spec.Inventory > 0) {
+			fmt.Println("Using dynamic inventory, setting existing inventory to nil")
+			// do something with pre-existing inventory state?
+		}
+
+		foundSource := lookupInventorySource("./tmp/", "inventory_source.yml")
```

Comment:
Instead of using ./tmp/ to store the inventory source, it might be worth to try if /runner/inventory/ directory works for the plugin source. Ansible runner uses convention over configuration within the contents of the /runner/ directory. I.E., if you put an inventory.yaml file in /runner/inventory, it will be used right away, no more configuration needed. Maybe that also happens with inventory sources for the plugins